extensions/wikia/WikiaApi/WikiaApiQueryAllUsers.php

```diff
@@ -263,55 +251,55 @@ private function local_users() {
 
 		# prop
 		$prop = $params['prop'];
-		if ( !is_null($prop) ) {
-			$prop = array_flip($prop);
+		if ( !is_null( $prop ) ) {
+			$prop = array_flip( $prop );
 		}
-		
+
 		# table
-		$this->addTables('events_local_users');
-		$this->addWhere( 'wiki_id = ' . intval($wgCityId) );
+		$this->addTables( 'events_local_users' );
+		$this->addWhere( 'wiki_id = ' . intval( $wgCityId ) );
 		$this->addWhere( 'user_is_closed = 0' );
-		
+
 		# limit
 		$limit = $params['limit'];
-		$this->addOption('LIMIT', $limit);		
-				
-		# params		
-		$fld_blockinfo 		= isset($prop['blockinfo']);
-		$fld_editcount 		= isset($prop['editcount']);
-		$fld_groups 		= isset($prop['groups']);
-		$fld_registration 	= isset($prop['registration']);
-
-		if ( !is_null($params['from']) )
-			$this->addWhere( 'user_name >= ' . $db->addQuotes($this->keyToTitle($params['from'])) );
-
-		if ( !is_null($params['prefix']) )
+		$this->addOption( 'LIMIT', $limit );
+
+		# params
+		$fld_blockinfo 		= isset( $prop['blockinfo'] );
+		$fld_editcount 		= isset( $prop['editcount'] );
+		$fld_groups 		= isset( $prop['groups'] );
+		$fld_registration 	= isset( $prop['registration'] );
+
+		if ( !is_null( $params['from'] ) )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)